server/models/Request.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user: { 
```

Comment:
Hi @BerryGIT-ME, Ajay was referring to adding a reference to the User model in the userId/sitterId fields.

Example: [{ type: Schema.Types.ObjectId, ref: 'Person' }]
Ref Link: https://mongoosejs.com/docs/populate.html

Reply:
Ok thanks, I'll make that update right now